bot/app/database/db_update.py

```diff
@@ -1,173 +1,83 @@
-"""
+from db_objects import Artist, Event, PlaysAt
 
-import sqlite3
-import logging
+def new_event():
 
-class Event:
+    e_name = input("Enter Event-name: ")
+    date = input("Enter date (format DD.MM.YYYY): ")
+    time = input("Enter event time: ")
+    admission = input("Enter admission: ")
+    desc = input("Enter description: ")
+    loca = input("Enter location name: ")
+    e_pic_id = input("Enter the PicID: ")
+    
+    event = Event(e_name, date, time, admission, desc, loca, e_pic_id)
+    event.insert_event()
 
-    """
-    An instance of the Event relation 
-    """
+def new_artist():
 
-    def __init__(self):
-        
-        self.conn = sqlite3.connect('data.sqlite')
-        
-        self.eName = input("Enter Event-name: ")
-        self.date = input("Enter date (format DD.MM.YYYY): ")
-        self.time = input("Enter event time: ")
-        self.admission = input("Enter admission: ")
-        self.desc = input("Enter description: ")
-        self.ePicID = input("Enter the PicID: ")
-
-        self.insert_event()
-        self.conn.close()
+    a_name = input("Enter the artists's name: ")
+    webs = input("Enter Website: ")
+    soundc = input("Enter Soundcloud account: ")
+    bandc = input("Enter Bandcamp Profile: ")
+    bio = input("Enter Bio: ")
+    a_pic_id = input("Enter the PicID: ")
     
-    def insert_event(self):
-        
-        curs = self.conn.cursor()
-
-        try:
-            curs.execute("""
-                insert into Events values (?,?,?,?,?,?)""",
-                (self.eName, self.date, self.time, self.admission, self.desc, \
-                self.ePicID))
-            self.conn.commit()
-        
-        except sqlite3.IntegrityError:
-            print("Event already exists in DB")
-            return
-        
-        curs.close()
-        
-class Artist:
-
-    def __init__(self):
+    artist = Artist(a_name, webs, soundc, bandc, bio, a_pic_id)
+    artist.insert_artist()
 
-        self.conn = sqlite3.connect('data.sqlite')
-        
-        self.aName = input("Enter the Artist's name: ")
-        self.webs = input("Enter Website: ")
-        self.soundc = input("Enter Soundcloud account: ")
-        self.bandc = input("Enter Bandcamp Profile: ")
-        self.bio = input("Enter Bio: ")
-        self.aPicID = input("Enter the PicID: ")
-
-        self.insert_artist()
-        
-        self.conn.close()
+def plays_at_relation():
 
-    def insert_artist(self):
-        
-        curs = self.conn.cursor()
-
-        try:
-            curs.execute("""
-                insert into Artists values (?,?,?,?,?,?)""",
-                (self.aName, self.webs, self.soundc, self.bandc, self.bio, \
-                self.aPicID))
-            self.conn.commit()
-        
-        except sqlite3.IntegrityError:
-            print("Artist already exists in DB")
-            return
-        
-        curs.close()
+    a_name = find_artist()
+    e_name, date = find_event()
```

Comment:
This throws an Error if find_event() returns none:
`TypeError: cannot unpack non-iterable NoneType object`
Not sure how to address this issue

Reply:
solved with `try/except block`